Index: src/main/java/com/example/hostelmanagementsystem/HostelManagementSystem.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.hostelmanagementsystem;\r\n\r\nimport javafx.application.Application;\r\nimport javafx.scene.Scene;\r\nimport javafx.scene.control.Label;\r\nimport javafx.scene.layout.BorderPane;\r\nimport javafx.stage.Stage;\r\n\r\nimport java.io.IOException;\r\n\r\npublic class HostelManagementSystem extends Application {\r\n    @Override\r\n    public void start(Stage stage) throws IOException {\r\n        BorderPane bp= new BorderPane();\r\n        Label title = new Label(\"Hello World\");\r\n        bp.setCenter(title);\r\n        Scene scene = new Scene(bp, 320, 240);\r\n        stage.setTitle(\"Hello!\");\r\n        stage.setScene(scene);\r\n        stage.show();\r\n    }\r\n\r\n    public static void main(String[] args) {\r\n        System.out.println(new Person());\r\n        System.out.println(new Lecturer());\r\n        System.out.println(new Student());\r\n        launch();\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/hostelmanagementsystem/HostelManagementSystem.java b/src/main/java/com/example/hostelmanagementsystem/HostelManagementSystem.java
--- a/src/main/java/com/example/hostelmanagementsystem/HostelManagementSystem.java	
+++ b/src/main/java/com/example/hostelmanagementsystem/HostelManagementSystem.java	
@@ -26,4 +26,4 @@
         System.out.println(new Student());
         launch();
     }
-}
\ No newline at end of file
+}
